PackageExplorer/Controls/PdbInfoViewer.xaml

```diff
@@ -63,7 +63,7 @@
             Grid.Column="1"
             Visibility="{Binding CompilerFlags, Converter={StaticResource NullToVisibilityConverter}}"
             ItemsSource="{Binding CompilerFlags}"
-            MaxHeight="250">
```

Comment:
@clairernovotny was there a reason why you did set a `MaxHeight`?

Reply:
The idea was that if the compiler list became too long, it'd scroll. That way the references wouldn't be too far down. Not sure on the best UX for the tab as a whole.